source/_integrations/zone.markdown

```diff
@@ -14,7 +14,6 @@ The `zone` integration uses YAML sequences to configure multiple zones:
 
 ```yaml
 # Example configuration.yaml entry
-zone:
```

Comment:
uh? This is not the zone file, as written on multiple places, even in the line above this one.